config.go

```diff
@@ -119,6 +119,16 @@ var (
 		lndDefaultConfig.DataDir, defaultLndChainSubDir,
 		defaultLndChain, DefaultNetwork, defaultLndMacaroon,
 	)
+
+	// DefaultMacaroonFilename is the default file name for the
+	// autogenerated lit macaroon.
+	DefaultMacaroonFilename = "lit.macaroon"
```

Comment:
IIRC, we'll need to also add these caveats to the super macaroon, correct?

Reply:
yes. the perms in `litPermissions` are already added to the super mac and it is populated in the next commit. (i think this is what you mean?)